client/apps/edit/components/content2/sections/header/index.jsx

```diff
@@ -136,6 +145,14 @@ export default class SectionHeader extends Component {
     } else {
       return (
         <div className={'edit-header' + headerClass}>
+          <div>
+            <div className='status'>
+              Status: {this.props.status}
+            </div>
+            <button onClick={() => this.props.helloWorldAction({ status: 'Hi! ' + Math.random() })}>
+              Click me to update status!
+            </button>
+          </div>
```

Comment:
Works great! Can we remove this before merging though? `content2` is being used as the default already which means it's visible to real people :P